src/explore-education-statistics-admin/src/pages/release/datablocks/__data__/tableToolServiceData.ts

```diff
@@ -6,25 +6,30 @@ import {
 export const testSubjectMeta: SubjectMeta = {
   filters: {
     Characteristic: {
+      id: 'characteristic',
       totalValue: '',
       hint: 'Filter by pupil characteristic',
       legend: 'Characteristic',
       name: 'characteristic',
       options: {
         Gender: {
+          id: 'gender',
           label: 'Gender',
           options: [
             {
               label: 'Gender female',
               value: 'gender-female',
             },
           ],
+          order: 0,
```

Comment:
In the previous front end work to add the reordering tab to the Admin we added an order field to the types that are objects in the API responses for subject meta data and table result subject meta data which is filters, filter groups and indicator groups. I think these corresponded with the 'legend options' view model types on the back end previously. The options (which will be the filter items or indicators) are arrays so we can infer the order from the array position. For the others I did this because I thought with them being key/values of an object that their position shouldn't be relied on for an order. A lot of the changes on this PR will be because I now made the order field mandatory whereas it was optional before.